src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java

```diff
@@ -0,0 +1,26 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VUAP;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+public class TestVUAP extends TemplateNoFirstLoad {
+  private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
+
+  @Test(
+          groups = "AdsUapDesktop"
+  )
+  public void testVideoPlayFlow() {
+    Page page = new Page("project43", "SyntheticTests/VUAP");
+    AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(page), DESKTOP_SIZE);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "ads" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)